### PR TITLE
OTT-100 : Get Advertiser list from VAST

### PR DESCRIPTION
**Pull request by PubMatic-OpenWrap**

### CODE REVIEW

File: adapters/vastbidder/vast_tag_response_handler.go [hunk 1]
```diff
@@ -186,27 +187,43 @@ func getAdElement(vast *etree.Element) *etree.Element {
...
+	version, err := strconv.ParseFloat(vastVer, 64)
+	if err != nil {
+		version = 2.0
+	}
+
+	advertisers := make([]string, 0)
+
+	switch int(version) {
+	case 2, 3:
+		for _, ext := range ad.FindElements(`./Extensions/Extension/`) {
```
Comment (PubMatic-OpenWrap) [inline adapters/vastbidder/vast_tag_response_handler.go:200]: Move strings to constants

File: adapters/vastbidder/vast_tag_response_handler.go [hunk 2]
```diff
@@ -186,27 +187,43 @@ func getAdElement(vast *etree.Element) *etree.Element {
...
-	var advertisers []string
-	for _, ext := range extensions {
-		for _, attr := range ext.Attr {
-			if attr.Key == "type" && attr.Value == "advertiser" {
-				for _, ele := range ext.ChildElements() {
-					if ele.Tag == "Advertiser" {
-						if nil == advertisers {
-							advertisers = make([]string, 0)
+func getAdvertisers(vastVer string, ad *etree.Element) []string {
+	version, err := strconv.ParseFloat(vastVer, 64)
```
Comment (PubMatic-OpenWrap) [inline adapters/vastbidder/vast_tag_response_handler.go:191]: use etree.findAttr

**Comment by PubMatic-OpenWrap** [comment]
@ShriprasadM 
Didn't change the function signature to use float as version. We can discuss this

@shalmali
didn't change the strings to constants.
There will be a lot of confusion because we are using similar strings like - 
`advertiser`
`Advertiser`
`./Advertiser`

Also, we are using strings and not constants for other functions in the same file.

**Comment by ShriprasadM** [comment]
Hello Murad,
Can you justify, why can't we change the function signature to use float as version ?

**Pull request merged**